packages/api/src/server.ts

```diff
@@ -136,6 +138,30 @@ export default (app: express.Application): void => {
     // default
     app.use(config.api.v2prefix, v2routes());
 
+    app.use((error, req, res, next) => {
+        utils.Logger.error(
+            req.originalUrl + ' -> ' + error &&
+                error.details &&
+                error.details.get('body') &&
+                error.details.get('body').details &&
+                error.details.get('body').details.length > 0 &&
+                error.details.get('body').details[0].message
+        );
+        if (error && error.toString().indexOf('Validation failed') !== -1) {
+            return res.status(400).json({
+                success: false,
+                error: {
+                    name: 'INVALID_PAYLOAD',
+                    message: 'invalid payloads',
+                    details: error.details.get('query')
+                        ? error.details.get('query').details
+                        : error.details.get('body').details,
+                },
+            });
+        }
+        next();
+    });
+
```

Comment:
I think this was removed before (by me, if I'm not mistaken because I didn't see it being used). What was the reason to add it back?

Reply:
ahhh I see. Makes sense.
then I think, you can remove the `utils.Logger.error` part, and just add a comment explaining why this is important.